File: src/main/java/io/github/cdgeass/editor/dom/element/MapperDescription.java
package io.github.cdgeass.editor.dom.element;

import com.intellij.util.xml.DomFileDescription;
import io.github.cdgeass.constants.StringConstants;
import io.github.cdgeass.editor.dom.element.mapper.Mapper;

/**
 * @author cdgeass
 * @since 2020-05-20
 */
public class MapperDescription extends DomFileDescription<Mapper> {

    public MapperDescription() {
        super(Mapper.class, StringConstants.MAPPER);
    }

}

File: src/main/kotlin/io/github/cdgeass/codeInsight/dom/description/MapperDescription.kt
package io.github.cdgeass.codeInsight.dom.description

import com.intellij.util.xml.DomFileDescription
import io.github.cdgeass.codeInsight.dom.element.Mapper

/**
 * @author cdgeass
 * @since 2021-03-18
 */
class MapperDescription : DomFileDescription<Mapper>(Mapper::class.java, "mapper")
